<launch>
  <!-- Load Kortex driver -->
    <!-- Load Kortex driver -->
    <include file="$(find kortex_gazebo)/launch/spawn_kortex_robot.launch">
      <arg name="start_rviz" value="true" />
      <arg name="arm" value="gen3_lite" />
    </include>


  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen">
    <param name="video_device" value="/dev/video4" />
    <param name="image_width" value="1920" />
    <param name="image_height" value="1080" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap" />
  </node>

  <node name="aruco_detector" pkg="kinova_5551_chess_arm" type="aruco_detector.py" output="screen" />

</launch>